models/public/yolof/README.md

```diff
@@ -0,0 +1,124 @@
+# yolof
+
+## Use Case and High-Level Description
+
+YOLOF is a simple, fast, and efficient object detector without FPN. Model based on ["You Only Look One-level Feature"](https://arxiv.org/abs/2103.09460) paper. It was implemented in PyTorch\* framework. For details see [repository](https://github.com/megvii-model/YOLOF). This model was pre-trained on [Common Objects in Context (COCO)](https://cocodataset.org/#home) dataset with 80 classes.
```

Comment:
please provide more detailed description. There are several models in the original repo with different backbones and other interesting features. We need to make clear which model we demonstrate here for customers.